pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -761,7 +761,15 @@ func (vs *VSphere) AttachDisk(vmDiskPath string, nodeName k8stypes.NodeName) (di
 			return "", "", err
 		}
 
-		scsiController = getSCSIController(vmDevices, vs.cfg.Disk.SCSIControllerType)
+		// Get VM device list
+		_, vmDevices, _, err := getVirtualMachineDevices(ctx, vs.cfg, vs.client, vSphereInstance)
+		if err != nil {
+			glog.Errorf("cannot get vmDevices for VM")
```

Comment:
Can we print or return err?
glog.Errorf("cannot get vmDevices for VM. err: %s", err)